### ISSUE TITLE
Description Colors are problematic

### ISSUE THREAD

**Issue by Omurolo** [superseva/fallout#60]
Sadly, as much as I like the colors you choose, one of my players have a bad eyesight problem and the color used primarly (#dedb8a) is really hard for him to read when the background color is the one for mods (#6d879f).

I tried changing it, but I'm sadly unable to.

What can I do? I tried to open the Source Code for the mods adding the <span style="color: #2b2301;"> (basically, black) and it works and he can read it just fine... but doing this every time... for every object... Is killing me...

Please help ç_ç

TL:DR
Light yellow on light blue is making for one of my players a struggle to read

**Comment by superseva** [comment]
Hi,
you can solve this by installing the "Custom Css" module [https://foundryvtt.com/packages/custom-css](https://foundryvtt.com/packages/custom-css)
You can set the background globally or allow only one player to set it for him self by pasting this:
```.editor-content{
background-color:black
}```

You can change elements by inspecting the console for their class and then use it in your Custom Css.

![RJxQ5MsJ9O](https://user-images.githubusercontent.com/4093045/236624453-b770b306-2bc2-4d8c-8921-15a40129d318.png)

**Comment by Omurolo** [comment]
> ```
> .editor-content{
> background-color:black
This actually worked flawlessy

> You can change elements by inspecting the console for their class and then use it in your Custom Css.
Now I'm wondering, what should I write to change the text color instead of the background?

Also, thank you for your time!

**Comment by superseva** [comment]
Try adding 
color:#ffffff to change text to white or just color:white

basically any css definition for the element in question will work.

**Comment by superseva** [comment]
use `;` to separate lines like this
```
.editor-content{
background-color:black;
color:white;
}
```

**Comment by superseva** [comment]
if the text is to small you can also put
font-size:1.5em; in there too.
or any other css definition

**Comment by Omurolo** [comment]
You are a life-saver! Thank you so very much!
You can close, this definitely solves every possible issues we might have!

**Comment by superseva** [comment]
Glad to help
